Project.toml

```diff
@@ -27,7 +27,7 @@ test = ["Base64", "DelimitedFiles", "Test"]
 
 [compat]
 julia = "1"
-ArnoldiMethod = "0.0.4"
-DataStructures = "0.17.9, 0.18"
-Inflate = "0.1.1"
-SimpleTraits = "0.9.1"
+ArnoldiMethod = "0.0"
```

Comment:
ArnoldiMethod v0.0.5 might have breaking changes from v0.0.4 if I interpret https://julialang.github.io/Pkg.jl/v1/compatibility/ correctly

Reply:
I did not realize that there were again special rules for the last digit...  @sbromberger if you are worried about this, you'll have to change it back.